### PR TITLE
Sharing: fix all PHPCS errors

### LINKED ISSUE #21970: PHPCS: Sharing
### Impacted plugin

Jetpack

### What

PHPCS

### How

- [ ] `projects/plugins/jetpack/modules/sharedaddy/recaptcha.php`
- [ ] `projects/plugins/jetpack/modules/sharedaddy/sharedaddy.php`
- [ ] `projects/plugins/jetpack/modules/sharedaddy/sharing-service.php`
- [ ] `projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php`
- [ ] `projects/plugins/jetpack/modules/sharedaddy/sharing.php`

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #21970

#### Changes proposed in this Pull Request:

* Fix PHPCS errors in all sharing files.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site where the Sharing feature is disabled.
* Go to Jetpack > Settings > Sharing and enable the feature.
* Go to Settings > Sharing and ensure that the 2 default buttons are displayed there.
* Enable the sharing buttons on single posts, save your changes, ensure the changes get saved.
* Ensure the buttons are displayed properly on the frontend.
* Ensure you can use the buttons properly.
* Now go back to Settings > Sharing and try to change some settings, add some buttons, add [custom sharing buttons](https://ryanmarkel.com/3004/adding-specific-sharing-services-to-sharedaddy-or-jetpack/). Ensure changes are always saved properly, and reflected on the frontend.
* Try editing some of your posts, and overwriting sharing button status there in the Jetpack sidebar. That should work too.
* Is CI happy?

### CODE REVIEW

**Review by coder-karen — CHANGES_REQUESTED** [review]
I left a few small suggestions.

Separately, I seem to be having an issue updating posts, only when sharing is enabled. Testing on JN with this feature branch. I get an 'updating failed' error message - and in the console I see  `POST https://near-hammerhead.jurassic.ninja/wp-json/wp/v2/posts/1?_locale=user 403.`
Prior to testing updating a post I had added a custom sharing button which worked fine. I removed it and it made no difference to the post update errors afterwards.
I created a new JN site to test again and had the same issue.

File: projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php [hunk 1]
```diff
@@ -226,22 +313,44 @@ public function get_link( $url, $text, $title, $query = '', $id = false ) {
...
 	 * @uses get_permalink
+	 *
 	 * @return string get_permalink( $post_id ) Post permalink.
 	 */
 	public function get_process_request_url( $post_id ) {
 		return get_permalink( $post_id );
 	}
 
+	/**
+	 * Get sharing name.
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php:327]: Sure thing. Would 7c0a8c54e2cb966e914410870b901769a45a60b5 work?

**Comment by jeherve** [comment]
> I get an 'updating failed' error message
 c5611e64a3df43a73ef5878a9bb3758d9c80b232 should fix this issue.

File: projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (coder-karen) [reply projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php:327]: Looks good!

**Comment by coder-karen** [comment]
I still notice the issue updating posts (and also an error appearing when initially clicking 'edit' or 'add new' in the main posts admin page - but only when testing the feature branch on JN.

**Comment by jeherve** [comment]
> I still notice the issue updating posts (and also an error appearing when initially clicking 'edit' or 'add new' in the main posts admin page - but only when testing the feature branch on JN.

I can't seem to reproduce at the moment. Is this when running the most recent version of the branch?

**Comment by coder-karen** [comment]
It was - I replicated again first thing just now, but after navigating around and toggling settings some more, it started working. Caching, possibly? Anyway seems to have self resolved.
🚢

**Review by coder-karen — APPROVED** [review]

**Comment by jeherve** [comment]
r246114-wpcom

**Comment by jeherve** [comment]
> It was - I replicated again first thing just now, but after navigating around and toggling settings some more, it started working. Caching, possibly? Anyway seems to have self resolved. 🚢

I reproduced as well just now, and filed #24483 to fix this.

**Pull request merged**